live-examples/js-examples/weakmap-prototype-has.html

```diff
@@ -0,0 +1,14 @@
+<pre>
+<code id="static-js">var weakmap1 = new WeakMap();
+var object1 = new Object();
+var object2 = new Object();
+
+weakmap1.set(object1, 42);
+
+console.log(weakmap1.get(object1));
+// expected output: 42
+
+console.log(weakmap1.get(object2));
+// expected output: undefined
+</code>
+</pre>
```

Comment:
This doesn't seem to be an example of `has`

Reply:
Yep - silly mistake my end. Duplicated previous example, wrote over it but didn't save for some reason. Should have picked this up in review. Apologies.